### PR TITLE
Elaborated logs about docker issues

### PR DESCRIPTION
**Pull request by jiivan**
fixes: #1416

### CODE REVIEW

**Comment by maaktweluit** [comment]
Relates to #1571

File: golem/core/processmonitor.py [hunk 1]
```diff
@@ -35,6 +35,10 @@ def _start(self):
                     if self.working:
                         self.run_callbacks(process)
                     self._child_processes.pop(i)
+                    if isinstance(process, (psutil.Popen, subprocess.Popen)):
+                        stdout_data, stderr_data = process.communicate()
+                        print("STDOUT: {}\nSTDERR: {}"
```
Comment (maaktweluit) [inline golem/core/processmonitor.py:40]: maybe use `logger` over `print`

`logger` has native formatting, works faster when the log level is turned off. Samples in #1571

File: golem/docker/manager.py [hunk 2]
```diff
@@ -409,6 +409,18 @@ def _set_docker_machine_env(self, retried=False):
...
+Ensure that you try the following before reporting an issue:
+
+ 1. The virtualization of Intel VT-x/EPT or AMD-V/RVI is enabled in BIOS
+    or virtual machine settings.
+ 2. The proper environment is set. On windows powershell please run:
+    & "C:\Program Files\Docker Toolbox\docker-machine.exe" env golem | Invoke-Expression
+ 3. virtualbox driver is available:
+    docker-machine.exe create --driver virtualbox golem
+ 4. Restart Windows machine"""
+            logger.error(typical_solution_s)
```
Comment (maaktweluit) [inline golem/docker/manager.py:422]: Is this problem also on macs? in that case we should also add a similar block there.

I can help test on mac if you have none available.

**Review by maaktweluit — CHANGES_REQUESTED** [review]
Good start! small comments

File: golem/docker/manager.py (hunk 2 above)
Comment (jiivan) [reply golem/docker/manager.py:422]: Windows instances are most problematic in this issue, but I will be glad if you could test it on mac, or have experienced any similar problems (usually on "fresh" installs).

**Comment by jiivan** [comment]
Closed in favor of #1588

File: golem/docker/manager.py (hunk 2 above)
Comment (maaktweluit) [reply golem/docker/manager.py:422]: Ok, will test it next week when back from holiday :) This is already a great addition to help the end users.

**Pull request closed without merging**